src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum Colors {
+    RED,
+    GREEN,
+    BlUE,
```

Comment:
Enum values should be written in uppercase letters. Please change 'BlUE' to 'BLUE'.